tests/__helpers__/tasksRepositoryMock.ts

```diff
@@ -0,0 +1,28 @@
+import * as taskRepository from '../../src/repositories/tasks';
+import { functionConfig, generateFunction } from './functions';
+
+export const tasksRepositoryMock = (update = false) => {
+  const existingFunction = generateFunction(functionConfig);
+  const findData = update ? [existingFunction] : [];
+
+  const findFunctionsSpy = jest.spyOn(taskRepository.functions, 'find')
+    .mockImplementation(() => Promise.resolve(findData));
```

Comment:
(minor/silly/not-blocking) This does not match what the end point actually returns.